doc/kernel_tests.rst

```diff
@@ -71,6 +71,10 @@ Base API
 Implemented tests
 =================
 
+The following tests are implemented:
+
+.. include:: test_list.rst
```

Comment:
That's pretty nifty! However this makes me wonder if we shouldn't change the hierarchy a bit. Maybe something like this?

```
* Introduction
* Writing tests
** Basics
** Where to start
(nothing changed up till now)

* Available tests
(this new fancy list)

* API
(base + implemented tests API)
```

Reply:
yes, I was also wondering if we should put that list a bit more under the spotlight somehow. A user may not necessarily be interested in the APIs directly, but more about "what can I run with lisa-test/exekall and what can I expect to test using that LISA software ?"